### ISSUE TITLE
Histogram creation after moving the .nxs files to a different folder causes an error.

### ISSUE THREAD

**Issue by MarcBHahn** [BAMresearch/McSAS3#8]
Hi,
I think I encountered some bug, or at least unexpected behavior during the histogram creation:

After fitting some datasets in "\~/fitting_folder/" and moving them to a different folder "\~/histogramming_folder/" the histogram creation with mcsas3_cli_histogrammer.py fails with the following errormessage:

```
...
assert self.filename.is_file(), f"input filename: {self.filename} must exist"
AssertionError: input filename:~/fitting_folder/input.dat must exist
```

This seems strange, that it needs the original ascii data from the ".csv/.dat" files, despite having all information needed to create plots and histograms in the ".nxs" files, which are provided as an input parameter to the histogrammer. Furthermore it still searchs in the "old" fitting_folder.

Bests
Marc

**Comment by toqduj** [comment]
Yes, that's a bug indeed, and it's related to the way it reconstructs its child objects from a saved file. 

I'll have to do some deep digging to fix that one.
